.travis.yml

```diff
@@ -1,9 +1,5 @@
 language: node_js
 node_js:
-  - '4.7'
-  - '4.8'
-  - '5.6'
-  - '5.7'
   - '6'
   - 'lts/*'
 # nodejs 11.11.X was failing with this https://travis-ci.org/moxystudio/node-proper-lockfile/jobs/503161746#L469
```

Comment:
Add ‘node’ to test latest stable node version, see https://github.com/moxystudio/stylelint-config-moxy/blob/cf536d49bb1ebf4e99d02ad764c107d8064c54f6/.travis.yml#L3